examples/reminders/lib/base/di/app_dependencies.dart

```diff
@@ -63,14 +65,14 @@ class AppDependencies {
         ..._interceptors,
       ];
 
-  List<Provider> get _analytics => [
-        Provider<FirebaseAnalytics>(
-            create: (context) => FirebaseAnalytics.instance),
-        Provider<FirebaseAnalyticsObserver>(
-          create: (context) =>
-              FirebaseAnalyticsObserver(analytics: context.read()),
-        ),
-      ];
+  // List<Provider> get _analytics => [
```

Comment:
Please remove the commented lines if they are not neede

Reply:
The comment lines were removed and a check was added for the `EnvironmentConfig` value